packages/block-library/src/navigation/edit/unsaved-inner-blocks.js

```diff
@@ -172,18 +171,9 @@ export default function UnsavedInnerBlocks( {
 	const Wrapper = isSaving ? Disabled : 'div';
 
 	return (
-		<Wrapper className="wp-block-navigation__unsaved-changes">
```

Comment:
There used to be an overlay to prevent direct manipulation, but I don't think this is needed anymore. It causes the styling of the inner blocks to be incorrect. There must be direct descendent selectors among the block styles, so I think it's best to try to match the DOM structure that the block normally has.

This could do with some testing, but I believe the unsaved inner blocks are save upon editing now, and it seemed to work ok for me. It's questionable if the `Disabled` wrapper is also still needed, but I left it there.

I did spot some issues that are also present in trunk, like the way links added after page list are aligned to the right:
![Screen Shot 2022-08-25 at 3 03 19 pm](https://user-images.githubusercontent.com/677833/186597623-0bf8ea89-0cdf-44c3-89d5-286ae31a081e.png)